src/blocks/music.rs

```diff
@@ -375,12 +383,14 @@ impl Player {
             player_proxy: proxy,
             title: metadata.title().map(Into::into),
             artist: metadata.artist().map(Into::into),
+            url: metadata.url().map(Into::into),
```

Comment:
I think these `.map(Into::into)` are not necessary anymore. Not directly related to your PR, but can you please remove them?

Reply:
Sure, fixed in 5217ccd28e2a32f522531fe531629596ee21571a.